src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+            try {
+                switch(input){
+                    case "list":
+                        print();
+                        break;
+                    case "mark":
+                        int markItem = sc.nextInt();
+                        taskList.get(markItem - 1).markDone();
+                        break;
+                    case "unmark":
+                        int unmarkItem = sc.nextInt();
+                        taskList.get(unmarkItem - 1).markUndone();
+                        break;
+                    case "delete":
+                         int deleteItem = sc.nextInt();
+                        delete(deleteItem);
+                        break;
+                    case "todo":
+                        add(new ToDo(sc.nextLine()));
+                        break;
+                    case "event":
+                        String eventCommand = sc.nextLine();
+                        String[] event = new String[3];
+                        event[0] = eventCommand.substring(1, eventCommand.indexOf(" /"));
+                        event[1] = eventCommand.substring(eventCommand.indexOf("/from") + 6, 
+                        eventCommand.indexOf(" /to"));
+                        event[2] = eventCommand.substring(eventCommand.indexOf("/to") + 4);
+                        add(new Event(event[0], event[1], event[2]));
+                        break;
+                    case "deadline":
+                        String ddlCommand = sc.nextLine();
+                        String[] ddl = ddlCommand.split(" /by ");
+                        add(new Deadline(ddl[0].substring(1), ddl[1]));
+                        break;
+                    default:
+                        add(new Task(input));
+                }
+            } 
+
+            catch (DukeException e) {
+                System.out.println(e.getMessage());
+            }
+
+            finally {
+                System.out.println(partition);
+                input = sc.next();
+            }
+        }
+            
+        exit();
+    }
+
+    public static void add(Task input) {
+        taskList.add(input);
+        System.out.println("Got it! This task has been added: ");
+        System.out.println(input.getStatus());
+        System.out.println("Current # of task: " + taskList.size());
+    }
+
+    public static void delete(int num) {
+        System.out.println("I've removed this task:");
+        taskList.get(num-1).getStatus();
+        taskList.remove(num - 1);
+        System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+    }
+
+    public static void print() {
+        int index = 1;
+        for (Task task: taskList) {
+            System.out.println(index + ". " + task.getStatus());
+            index ++;
+        }
+    }
+
+    public static void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(partition);
     }
 }
```

Comment:
Indentation & Formatting:

The code seems to be properly indented, which makes it easier to read.
The consistent use of brackets {} and their placement adheres to the Java coding conventions.
Variable Naming:

Names like input, eventCommand, and ddlCommand are meaningful, but some variables like ddl could have a more descriptive name.
Variables should be named in a manner that makes their purpose clear. For instance, ddl might be better named as deadlineDetails or similar.
sc has not been defined in the given snippet, so it's unclear what it represents. If it's a Scanner object, a more intuitive name like scanner might be more appropriate.